library/image_streamer_artifact_bundle.py

```diff
@@ -262,24 +234,14 @@ def __present(self, data, resource):
             changed, msg, facts = self.__create(data)
         else:
             changed = False
-            msg = ARTIFACT_BUNDLE_ALREADY_EXIST
+            msg = self.MSG_ALREADY_EXIST
             facts = dict(artifact_bundle=resource)
         return changed, msg, facts
 
-    def __absent(self, resource):
-        if resource:
-            self.i3s_client.artifact_bundles.delete(resource)
-            changed = True
-            msg = ARTIFACT_BUNDLE_DELETED
-        else:
-            changed = False
-            msg = ARTIFACT_BUNDLE_ABSENT
-        return changed, msg, {}
-
     def __create(self, data):
         data['buildPlans'] = [data['buildPlans']]
```

Comment:
I believe it should be fixed on the example, not here.
buildPlans, planScripts, deploymentPlans, and goldenImages are documented as arrays of InputArtifacts.